.github/workflows/main.yml

```diff
@@ -218,7 +222,7 @@ jobs:
           file: ./Dockerfile
           platforms: linux/amd64,linux/arm64
           push: true
-          tags: ${{ github.repository }}:${{ needs.ci.outputs.PUBLISH_TAG }},${{ github.repository }}:${{ steps.slug.outputs.SHA7 }},${{ github.repository }}:${{ needs.ci.outputs.NPM_VERSION }}
+          tags: ${{ github.repository }}${{ needs.ci.outputs.IMAGE_SUFFIX }}:${{ needs.ci.outputs.PUBLISH_TAG }},${{ github.repository }}${{ needs.ci.outputs.IMAGE_SUFFIX }}:${{ steps.slug.outputs.SHA7 }},${{ github.repository }}${{ needs.ci.outputs.IMAGE_SUFFIX }}:${{ needs.ci.outputs.NPM_VERSION }}
```

Comment:
Should pull ` ${{ github.repository }}${{ needs.ci.outputs.IMAGE_SUFFIX }}` into a var as well, it's used in multiple places.